packages/devextreme/scss/widgets/fluent/timeView/_colors.scss

```diff
@@ -6,9 +6,9 @@
 $clock-digits-color: null !default;
 
 @if $mode == "light" {
-  $clock-digits-color: color.change($base-text-color, $alpha: 0.54) !default;
+  $clock-digits-color: $base-text-color !default;
```

Comment:
You don't need to concritize for 'light', 'dark' color themes. Let's leave a single variable

Reply:
fixed